### PR TITLE
add sentry provider

### PR DESCRIPTION
**Pull request by henriiik**
This PR adds a sentry provider for pulumi based on the terraform bridge.

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```
Comment (ringods) [inline CODE-OF-CONDUCT.md:1]: Can you remove this file? We have a global [Code of Conduct](https://github.com/pulumiverse/.github/blob/main/CODE_OF_CONDUCT.md) file in the [special `.github` organization repository](https://github.com/pulumiverse/.github/blob/main/CODE_OF_CONDUCT.md). The Github documentation doesn't have a clear overview of what can go into the special `.github` repository, but [I found it on StackOverflow](https://stackoverflow.com/a/61301254/7269988).

This global file automatically shows up in the sidebar on the right of every repository. Having a single one promotes consistency over all the repositories.

**Review by ringods — CHANGES_REQUESTED** [review]
Do not add tags yet as a way to try and release your provider. We are trying to get hold of one required package repository account. Once this is resolved, I will inform you.

File: CODE-OF-CONDUCT.md (hunk 1 above)
Comment (henriiik) [reply CODE-OF-CONDUCT.md]: fixed!

**Comment by henriiik** [comment]
> Do not add tags yet as a way to try and release your provider. We are trying to get hold of one required package repository account. Once this is resolved, I will inform you.
Got it!

**Comment by ringods** [comment]
@henriiik this repository has been equipped with the following secrets:

```
NPM_TOKEN (for npmjs.org)
NUGET_PUBLISH_KEY (for nuget.org)
PYPI_API_TOKEN (for pypi.org)
RELEASE_GITHUB_TOKEN (for GoReleaser to create a Github Release)
```

You should be able to start releasing this provider.

**Comment by henriiik** [comment]
@ringods thanks!

**Pull request merged**